app/imports/server/api/users.test.js

```diff
@@ -0,0 +1,67 @@
+import { Meteor } from 'meteor/meteor';
+import { chai } from 'meteor/practicalmeteor:chai';
+import { users } from '../../../server/api/users.js';
+
+var assert = chai.assert,
+    expect = chai.expect;
+
+const VALID_DOC = {
+      profile: {
+        name: 'Ricky Ng-Adam', 
+        email: 'rngadam@example.com'
+      },
+      createdAt: new Date()
+    };
+
```

Comment:
add test to confirm valid doc remains valid ( maybe the schema changes later on)